OpenSim/Common/Test/DataTable.cpp

```diff
@@ -2567,5 +2567,25 @@ int main() {
 
     test8();
 
+    std::vector<SimTK::Real> tsdt_data{1, 1, 1, 1, 
+                                       2, 2, 2, 2, 
+                                       3, 3, 3, 3};
+    OpenSim::TimeSeriesDataTable_<SimTK::Real, float> 
+        tsdt{tsdt_data.cbegin(),
+             tsdt_data.cend(),
+             4,
+             OpenSim::TraverseDir::RowMajor};
+
+    std::vector<double> tmp{4, 4, 4, 4};
+    tsdt.addRow(tmp);
+
+    std::cout << tsdt.getNumRows() << std::endl;
+
+    std::vector<std::string> tsdt_labels{"col-one", "col-two", "col-three", 
+                                         "col-four"};
+    tsdt.setColumnLabels(tsdt_labels);
+
+    tsdt.addTimestamps(std::vector<double>{1, 2, 3});
```

Comment:
Will this work with just the initializer list, since that is also a container? Using "time" rather than "timestamp" it would then be
```
tsdt.addTimes({1,2,3});
```
which is very compact.